doc/reference/power_management/index.rst

```diff
@@ -124,6 +124,27 @@ have higher wake latencies. Following is a thorough list of available states:
 .. doxygenenumvalue:: PM_STATE_SOFT_OFF
    :project: Zephyr
 
+.. _pm_constraints:
+
+Power States Constraint
+=======================
+
+The power management subsystem allows different Zephyr components and
+applications to set constraints on various power states preventing the
+system to go these states. This can be used by devices when executing
+tasks in background to avoid the system to go to state where it would
+lose context. Constraints can be set, released and checked using the
+follow APIs:
```

Comment:
Is this also valid in case of `pm_force_suspend_devices` is used ?
If not, maybe it should be also described here.

Reply:
These are system constraint and not device constraints, right now they are not explicitly connected, I mean, devices don't look system constraints and the system don't look devices constraints. `pm_force_suspend_devices` suspend devices independently of the system constraint

Reply:
From my point of view (and from what I recall from our discussions) devices should also be able to put constraints.
As for instance a some device may accept to work in sleep or standby modes and we need to be able to assert that.
I'm about to push a new version of the PM changes for stm32 drivers, this will be the opportunity to discuss that.

Reply:
yep, you're right, device can set constraints. Is this not clear in that text ? If it is not I can change it to be more clear.

Reply:
To me, what would deserve to be clarified is the impact vs `pm_force_suspend_devices`.

Reply:
Sorry, but I don't see the point here. These are system constraints and `pm_force_suspend_devices` is orthogonal to this. I've cited there that devices can put constraints. From this API and system perspective, does not matter if it was a device or the application that set the constraint, the behavior is the same.

Reply:
Ok